### PR TITLE
docs: code of conduct

### PR DESCRIPTION
**Pull request by ematipico**
**What kind of change does this PR introduce?**
No
**Did you add tests for your changes?**
No

**If relevant, did you update the documentation?**
No

**Summary**
Added a simple template for code of conduct. Copied from webpack.js.org

**Does this PR introduce a breaking change?**

No

**Other information**

### CODE REVIEW

**Comment by ematipico** [comment]
Github scans the `CODE_OF_CONDUCT.md` file inside the root folder of the project

**Comment by anshumanv** [comment]
Apparently, it scans .github too - https://help.github.com/en/articles/adding-a-code-of-conduct-to-your-project#adding-a-code-of-conduct-manually 😄  fine either way ✨

**Comment by ematipico** [comment]
I think it's more visible if we left it in the project root. What do you think? It's not a .github thing, it's a project thing.

**Comment by anshumanv** [comment]
Yeah lgtm, you think linking it in the README will prove to be helpful? 🤔

**Comment by ematipico** [comment]
> Yeah lgtm, you think linking it in the README will prove to be helpful? 

@anshumanv Maybe from the contribution readme file?

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Code of Conduct
+---------------
+
+At webpack and all webpack/webpack-contrib repositories we follow the
```
Comment (rishabh3112) [inline CODE_OF_CONDUCT.md]: Maybe I think it should be specific to `webpack-cli` only as this is for `webpack-cli`.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+[JSFoundation Code of Conduct][1]. Please adhere to the guidelines there and
+feel free to report any violation of them to the @webpack/core-team,
+@webpack/documentation-team, or <conduct@js.foundation>.
```
Comment (rishabh3112) [inline CODE_OF_CONDUCT.md]: And `@webpack/cli-team` maybe added here.

**Pull request merged**